orc8r/gateway/python/magma/magmad/rpc_servicer.py

```diff
@@ -344,3 +354,15 @@ def create_probe_protos(probes):
         ) for param in traceroute_param_protos]
         traceroute_results = traceroute.traceroute(traceroutes_to_exec)
         return map(create_result_proto, traceroute_results)
+
+    def _print_grpc(self, message):
```

Comment:
I think we should pull this function into `magma.common.rpc_utils` maybe? Could be somewhere else, but we seem to have this function in all the services.